docs/guide/mocking.md

```diff
@@ -673,14 +700,22 @@ test('can feed dogs', () => {
 ```
 :::
 
-Now, when we create a new instance of the `Dog` class its `speak` method (alongside `feed`) is already mocked:
+Now, when we create a new instance of the `Dog` class its `speak` method (alongside `feed` and `greet`) is already mocked:
 
 ```ts
 const dog = new Dog('Cooper')
```

Comment:
let's rename him to `Cooper` so we don't need to have numbers (`dog2`) later on

Reply:
Done 👍